lib/category_badge.rb

```diff
@@ -91,6 +91,6 @@ def self.style_for_email(category, opts)
     result << ERB::Util.html_escape(category.name)
     result << "</span></span>"
 
-    wrap_in_link(result, data[:category_url], "", "color: ##{category.text_color};")
+    wrap_in_link(result, data[:category_url], "")
```

Comment:
`extra_classes ` arg is optional too:

```suggestion
    wrap_in_link(result, data[:category_url])
```